src/python/pants/backend/docgen/tasks/confluence_publish.py

```diff
@@ -63,6 +61,12 @@ def api(self):
     return 'confluence1'
 
   def execute(self):
+    deprecated_conditional(
+      lambda: True,
+      '1.6.0.dev0',
+      'pants.backend.docgen.tasks.confluence_publish.py is deprecated',
```

Comment:
I don't believe you need the `is deprecated` bit, but it'd be good to manually test this in a context that triggered the `deprecated_conditional` to see exactly what the error this renders is.